src/vstart.sh

```diff
@@ -941,7 +941,7 @@ then
   while [ true ]
   do
     CEPH_PORT="$(echo $(( RANDOM % 1000 + 40000 )))"
-    ss -a -n | egrep ":${CEPH_PORT} .+LISTEN" 1>/dev/null 2>&1 || break
+    ss -a -n | egrep "\<LISTEN\>.+:${CEPH_PORT}\s+" 1>/dev/null 2>&1 || break
```

Comment:
nit, s/ramdon/random/ in the commit message.

Reply:
we are close. please s/randon/random/.

Reply:
Done